config/profiles.json

```diff
@@ -1730,7 +1730,7 @@
             "resourceTypes": {
                 "Microsoft.Kusto": {
                     "management": {
-                        "2019-09-07": ["*"]
+                        "2019-11-09": ["*"]
                     }
                 }
             },
```

Comment:
Please take a check if there is any other services need to be updated in `profiles.json`